breguet_range.py

```diff
@@ -0,0 +1,62 @@
+from gpkit.shortcuts import *
+from gpkit import Variable, Model, units
+import numpy.testing as npt
+
+class  B_Range(Model):
+
+	"""Breguet Range Model
```

Comment:
would be good to state the assumptions of the model in the docstring -- e.g. fuel consumption varies linearly with thrust (via TSFC), and is independent of velocity.